cypress/integration/cucumber/step_definitions/login-test.js

```diff
@@ -0,0 +1,37 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+/// <reference types="Cypress" />
+
+beforeEach(() => {
+  cy.initiateExceptionsLogger();
+});
+
+afterEach(() => {
+  cy.end();
+});
+
+Given('User opens the website at {string}', (url) => {
+  cy.visit(url);
+});
+
+When('User enters username {string}', (username) => {
+  cy.get('#username').type(username);
+});
+
+And('User clicks {string}', (text) => cy.get('.iw0iHcFjY--gDZi9jUQ1_').click());
```

Comment:
what element are you retriving by `.iw0iHcFjY--gDZi9jUQ1_` ensure you use a more readable and unchanging retrieval mechanism

Reply:
Yeah, I thought so too. I'm still trying to figure out how to completely eliminate the classes for query selectors. A work around could be... Adding custom attributes to the components or tags. Like data-id's. This will mean I'll have to edit the targeted code base for the test to pass in deployment.